src/keylessBackup/encryption.test.ts

```diff
@@ -0,0 +1,66 @@
+import crypto from 'crypto'
+import {
+  decryptPassphrase,
+  deriveKeyFromKeyShares,
+  encryptPassphrase,
+  getSecp256K1KeyPair,
+} from './encryption'
+
+describe("Encryption utilities using Node's crypto and futoin-hkdf", () => {
+  let keyshare1: Buffer
+  let keyshare2: Buffer
+  let passphrase: string
+
+  beforeEach(() => {
+    keyshare1 = crypto.randomBytes(16)
+    keyshare2 = crypto.randomBytes(16)
```

Comment:
I suggest using fixed constants here instead. Wherever possible, we should not use test inputs that vary to avoid inconsistent results.